<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMapBackdropOptions</class>
 <widget class="QMainWindow" name="frmMapBackdropOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>467</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>SWMM Map/Backdrop Options</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="cmdOK">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdCancel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbxMap">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>511</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Map Dimensions</string>
    </property>
    <widget class="QGroupBox" name="gbxLLMap">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Lower Left</string>
     </property>
     <widget class="QLabel" name="lblLLX">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X-Coordinate</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblLLY">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y-Coordinate</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtLLXMap">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtLLYMap">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gbxURMap">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Upper Right</string>
     </property>
     <widget class="QLabel" name="lblURX">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X-Coordinate</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtURYMap">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtURXMap">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblURY">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y-Coordinate</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="lblBackdrop">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Backdrop File Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBackdropFile">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbxMapUnits">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>511</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Map Units</string>
    </property>
    <widget class="QRadioButton" name="rbnFeet">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Feet</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbnMeters">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Meters</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbnDegrees">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Degrees</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbnNone">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbxBackdrop">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>511</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Backdrop Dimensions</string>
    </property>
    <widget class="QGroupBox" name="gbxLLBack">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Lower Left</string>
     </property>
     <widget class="QLabel" name="lblLLXBack">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X-Coordinate</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblLLYBack">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y-Coordinate</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtLLXBack">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtLLYBack">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gbxURBack">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Upper Right</string>
     </property>
     <widget class="QLabel" name="lblURXBack">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X-Coordinate</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtURYBack">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtURBack">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblURYBack">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y-Coordinate</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
